__tests__/app.test.js

```diff
@@ -81,4 +81,32 @@ describe('/api/articles/:article_id', () => {
             expect(response.body.msg).toBe('Bad request');
           });
       });
+})
+describe('/api/articles', () => {
+    test('GET:200 sends an array of article objects to the client', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            expect(response.body.articles.length).toBe(13)
+            response.body.articles.forEach((article) => {
+                expect(typeof article.article_id).toBe('number')
+                expect(typeof article.title).toBe('string');
+                expect(typeof article.topic).toBe('string');
+                expect(typeof article.author).toBe('string')
+                expect(typeof article.created_at).toBe('string')
+                expect(typeof article.votes).toBe('number')
+                expect(typeof article.article_img_url).toBe('string')
+                expect(typeof article.comment_count).toBe('string')
+            })
```

Comment:
You ideally also want to assert that the body property is NOT on the object. Have a look at how you may assert this: 
https://jestjs.io/docs/expect#expectnotobjectcontainingobject